src/inject/linux/index.js

```diff
@@ -0,0 +1 @@
+import './silent_Notification';
```

Comment:
Could change this to be `import '../darwin/slient_Notification';  Then we don't need to copy paste